apps/demo/src/migration-wizard/MigrationWizard.tsx

```diff
@@ -4,6 +4,11 @@ import { ConnectStep } from "./steps/connect/ConnectStep";
 import { DiscoveryStep } from "./steps/discovery/DiscoveryStep";
 import { useComputedHeightFromPageHeader } from "./hooks/UseComputedHeightFromPageHeader";
 import { useDiscoverySources } from "./contexts/discovery-sources/Context";
+import { PrepareMigrationStep } from "./steps/prepare-migration/PrepareMigrationStep";
+
+const openAssistedInstaller = (): void => {
+  window.open("https://console.dev.redhat.com/openshift/assisted-installer/clusters/~new?source=assisted_migration", "_blank");
```

Comment:
I think at the end of the day the migration wizard will be part of the `console.redhat.com`, in that case should we open new tab, or just forward current tab to new page?

Reply:
We discuss that in slack with Danielle and Yuval and we decide to open a new tab.